doc_src/gear_developers/limitation.md

```diff
@@ -0,0 +1,12 @@
+# Limitation
+
+There are certain limitations in Antiktyera.
+We introduce some main limitations. But all limitations are not described here. Please also read other parts of the document when you implement gear.
```

Comment:
Is my understanding correct?

```suggestion
We introduce some major limitations here. But this does not cover all the limitations of Antikythera. Please also read other parts of the documentation when you implement gear.
```

Reply:
Yes, it is.

Reply:
I fixed in 5611bc2eaeb9489572dfe9b50ac45127f5d06969